documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -139,6 +141,12 @@ in the `sample-weblogic-operator-ns` namespace,
 configures a deployment and supporting resources for the operator,
 and deploys the operator.
 
+In addition, it also configures a deployment and supporting resources for the 
```

Comment:
This seems a little buried to me. I think this would fit more clearly as a note right below the `Before installing the operator, ` note.  Then reword "In addition, it" to something like "By default, installing the operator".